lib/matplotlib/backends/backend_pdf.py

```diff
@@ -182,8 +183,8 @@ def pdfRepr(obj):
     # represented as Name objects.
     elif isinstance(obj, dict):
         r = [b"<<"]
-        r.extend([Name(key).pdfRepr() + b" " + pdfRepr(val)
-                  for key, val in six.iteritems(obj)])
+        r.extend([Name(key).pdfRepr() + b" " + pdfRepr(obj[key])
+                  for key in sorted(six.iterkeys(obj))])
```

Comment:
The `iterkeys` seems unnecessary if you're going to sort it anyway; `sorted(obj)` should work just fine.